lib/question.rb

```diff
@@ -16,6 +16,15 @@ def ask_non_whitespaces(question, field_name)
   end
 
   def ask_base(question)
-    ask("[?] #{question}")
+    ask("[?] #{question}") do |q|
+      yield(q) if block_given?
+    end
+  end
+
+  def required_with_min_length(question, field_name, min_length = 10)
```

Comment:
I don't know. I guess this goes along with the/a product definition. The spec doesn't include this. What should it be?